ru.json

```
@@ -0,0 +1,181 @@
+{
+	"footer": {
+		"description": "Modrinth — сервис с открытым исходным кодом.",
+		"copyright": "© Guavy LLC",
+		"legal": {
+			"heading": "Правовой материал",
+			"terms": "Условия использования",
+			"privacy": "Политика конфиденциальности",
+			"rules": "Правила"
+		},
+		"resources": {
+			"heading": "Ресурсы",
+			"blog": "Блог",
+			"docs": "Документация"
+		},
+		"buttons": {
+			"privacy": "Настройки конфиденциальности",
+			"theme": {
+				"label": "Изменить тему",
+				"system": "Тема системы",
+				"light": "Светлая тема",
+				"dark": "Тёмная тема",
+				"oled": "OLED-тема"
+			},
+			"language": {
+				"label": "Выберите язык",
+				"en": "Английский",
+				"en-GB": "Английский (Соединённое Королевство)",
+				"ru": "Русский (Россия)"
+			}
+		}
+	},
+	"error": {
+		"status": "Ошибка {status}",
+		"message": {
+			"404": "Ой-ёй, похоже вы заблудились в лабиринте.<br/>Страница, которую вы ищете, не найдена.",
+			"project_missing": "{type}, который вы ищете, не найден.",
+			"default": "Произошла непредвиденная ошибка"
+		},
+		"actions": {
+			"home": "Вернуться на главную",
+			"discord": "Discord"
+		}
+	},
+	"pages": {
+		"description": "Описание",
+		"gallery": "Галерея",
+		"changelog": "Список изменений",
+		"versions": "Версии",
+		"settings": "Настройки",
+		"reports": "Жалобы",
+		"moderation": "Модерация",
+		"notifications": "Уведомления",
+		"about": "О проекте",
+		"following": "Подписки",
+		"all": "Всё"
+	},
+	"generic": {
+		"labels": {
+			"license": "Лицензия",
+			"project_id": "ID проекта",
+			"project_status": "Статус проекта"
+		},
+		"actions": {
+			"edit": "Редактировать",
+			"follow": "Подписаться",
+			"unfollow": "Отписаться",
+			"report": "Пожаловаться",
+			"new_project": "Новый проект",
+			"download": "Скачать",
+			"save": "Сохранить",
+			"cancel": "Отмена",
+			"discard": "Сбросить"
+		},
+		"external": {
+			"github_profile": "Профиль на GitHub",
+			"discord": "Discord",
+			"github": "GitHub",
+			"issues": "Баг-трекер",
+			"source": "Исходный код",
+			"wiki": "Вики",
+			"patreon": "Patreon",
+			"paypal": "PayPal",
+			"buy_me_a_coffee": "Buy Me a Coffee",
+			"github_sponsors": "GitHub Sponsors",
+			"donate": "Пожертвовать",
+			"kofi": "Ko-Fi"
+		},
+		"environments": {
+			"label": "Окружение",
+			"server_side": "Сервер",
+			"client_side": "Клиент",
+			"values": {
+				"required": "Требуется",
+				"unsupported": "Не поддерживается",
+				"optional": "Необязательно",
```

```suggestion
				"optional": "Опционально",
```
Используется в обозначении рабочей среды мода/модпака, этот вариант уместнее.